_layouts/default.liquid

```diff
@@ -14,7 +14,7 @@
     {% include header.liquid %}
 
     <!-- Content -->
-    <div class="container mt-5" role="main">
```

Comment:
This role here is used for [accessibility purposes](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/main_role). I believe it should be kept.

Reply:
ok. I will revert the changes in the default.liquid.